### ISSUE TITLE
[REGRESSION] npm install playwright@0.12.0

### ISSUE THREAD

**Issue by jperl** [microsoft/playwright#1513]
**Context:**
- Playwright Version: 0.12.0
- Operating System: Mac
- Last working version: 0.11.1-next.1584997668668 (current next)

**Code Snippet**

See the `npm install` step: https://github.com/qawolf/playwright-video/runs/531356245

Checkout https://github.com/qawolf/playwright-video/pull/16/commits/71d7d372b4db7f55a694902ba027ab69b28c8ebc and run `npm install`

**Describe the bug**

```
(node:2762) UnhandledPromiseRejectionWarning: TypeError: downloadOptionsFromEnv is not a function
    at /home/runner/work/playwright-video/playwright-video/node_modules/playwright/install.js:21:40
    at Object.<anonymous> (/home/runner/work/playwright-video/playwright-video/node_modules/playwright/install.js:24:3)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
(node:2762) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
```

**Comment by mxschmitt** [comment]
I can also reproduce this behaviour:

```
➜  backend git:(renovate/all-minor-patch) ✗ npm install playwright-core@0.12.0 playwright@0.12.0

> playwright@0.12.0 install /Users/max.schmitt/Development/try-playwright/backend/node_modules/playwright
> node install.js

(node:4819) UnhandledPromiseRejectionWarning: TypeError: downloadOptionsFromEnv is not a function
    at /Users/max.schmitt/Development/try-playwright/backend/node_modules/playwright/install.js:21:40
    at Object.<anonymous> (/Users/max.schmitt/Development/try-playwright/backend/node_modules/playwright/install.js:24:3)
    at Module._compile (internal/modules/cjs/loader.js:971:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1011:10)
    at Module.load (internal/modules/cjs/loader.js:822:32)
    at Function.Module._load (internal/modules/cjs/loader.js:730:14)
    at Function.Module.runMain (internal/modules/cjs/loader.js:1051:12)
    at internal/main/run_main_module.js:16:11
(node:4819) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:4819) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
+ playwright@0.12.0
+ playwright-core@0.12.0
updated 2 packages and audited 1204719 packages in 11.879s
found 1503 low severity vulnerabilities
  run `npm audit fix` to fix them, or `npm audit` for details
```

**Comment by mxschmitt** [comment]
Will be fixed in #1514

**Comment by jperl** [comment]
I can confirm this is fixed in 0.12.1.
